explorer/app/hooks/useFetchEntities.ts

```diff
@@ -1,15 +1,20 @@
 import { isDevelopment } from "app/shared/constants";
 import { ListModel } from "app/models/viewModels";
 import { ListResponseType } from "app/models/responses";
-import { list } from "app/entity/api/service";
-import { useEffect } from "react";
+import { fetchList, list } from "app/entity/api/service";
+import { useCallback, useEffect, useState } from "react";
 import { useAsync } from "./useAsync";
 import { useCurrentEntity } from "./useCurrentEntity";
 import { isSSR } from "app/utils/ssr";
 
 interface UseEntityListResponse {
   response?: ListResponseType;
   isLoading: boolean;
+  nextPage?: () => void;
```

Comment:
Would it be possible to create a `Pagination` interface that encapsulates the `nextPage?`, `previousPage?`, `currentPage` e.t.c. values, and then add just add `pagination?` to `UseEntityListResponse`? I feel this would make the truly required pagination values more obviously required (e.g. `canNextPage` and `canPreviousPage`) and also makes a clearer truth check when configuring the React Table (e.g. `!!pagination` instead of `currentPage`) in  `table.tsx` (see [here](https://github.com/clevercanary/data-browser/blob/pagination-%2374/explorer/app/components/Table/table.tsx#L56)).

Reply:
great idea !!